docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
     - "80:80"
     networks:
       - front-net
+    container_name: reverse_proxy-container
```

Comment:
I think hardcoding the container name is not a good idea.
currently , you can scale this project using compose command
```bash
$ docker-compose scale bank_api=5
```
but, this command will not work due to this hardcoded container name.

Reply:
I have removed these hardcoded container names

Reply:
thanks!